test/performance/README.md

```diff
@@ -49,6 +49,58 @@ if err != nil {
 }
 ```
 
+## Zipkin trace
+
+Zipkin tracing can be enabled if needed by the performance test during setup.
+
+```
+perfClients, err := Setup(t, EnableZipkinTracing)
+```
+
+Once it is setup, all requests made by the `SpoofingClient` can be logged and saved to a trace file using the
+[AddTrace()](https://github.com/knative/serving/blob/master/test/performance/performance.go) method.
+
+Sample Trace:
+
+```
+...
```

Comment:
Update this to just include the trace json. Since, this mentions logging it into the trace file and not stdout.

Reply:
Updated.